core/templates/dev/head/domain/utilities/UrlInterpolationService.js

```diff
@@ -67,7 +67,7 @@ oppia.factory('UrlInterpolationService', ['alertsService',
 
       // Parameter values can only contain alphanumerical characters, spaces,
       // hyphens or underscores.
-      var VALID_PARAMETER_VALUE_REGEX = /^(\w| |_|-)+$/;
+      var VALID_PARAMETER_VALUE_REGEX = /^(\w| |_|-|\[|\]|'|")+$/;
```

Comment:
Having said that, I'm not a fan of this change. Can we instead have templates be just the main part of the URL (before the '?') with a more restricted regex? If we need to pass params, there's an angular API for that as part of $http.get: https://docs.angularjs.org/api/ng/service/$http#get